### ISSUE TITLE
Add 2-3 secured implementations for XXE vulnerability

### ISSUE THREAD

**Issue by preetkaran20** [SasanLabs/VulnerableApp#410]
**Is your feature request related to a problem? Please describe.**
As main target for Vulnerable App is scanners so we need to find out false positives by Scanners such that they can improve on there scan rules. So in order to find out the false positives, we need to add more secured implementations.

**Describe the solution you'd like**
Add 2-3 more functions in https://github.com/SasanLabs/VulnerableApp/blob/master/src/main/java/org/sasanlabs/service/vulnerability/xxe/XXEVulnerability.java which are secure.

**Comment by Prince-1110** [comment]
If this issue is not resolved yet, can I work on it?

**Comment by preetkaran20** [comment]
@Prince-1110  yes, you can work on it. I think there are few secure implementations of this vulnerability present in the class but not marked as secure so you can mark them as secure as well.
